lib/pg_ldap_sync/application.rb

```diff
@@ -85,6 +85,40 @@ def search_ldap_users
     return users
   end
 
+  def load_group_members_by_range(group_dn)
+    range_start = 0
+    all_group_members = []
+    while true do
+      member_size = 0
+      member_attribute_with_range = "#{@config[:ldap_groups][:member_attribute]}=#{range_start}-*"
+      returned_member_attribute_with_range = ""
+      log.debug " current attribute for range retrieval ----> #{member_attribute_with_range} "
+
+      res_entry = @ldap.search(
+        base:        group_dn,
+        filter:      Net::LDAP::Filter.eq('distinguishedName', group_dn), 
+        attributes:  member_attribute_with_range).first
+
+      res_entry.each do |attribute, values|
+        next if "#{attribute}" == "dn"
+        returned_member_attribute_with_range = "#{attribute}"
+        log.debug "  returned attribute --------> #{returned_member_attribute_with_range}"
+        member_size = values.count
+        values.each do |value|
+          log.debug "                              -----> #{value}"
```

Comment:
The member info is as useful as "user-dn", should this info be logged as "info" level?

Reply:
Yeah. I think it's reasonable to set the member info of a group to "info" level.
I will change it in next commit.
Thanks for your suggestion.